be/src/exprs/expr_context.h

```diff
@@ -165,11 +165,6 @@ class ExprContext {
     /// and owned by this ExprContext.
     std::vector<FunctionContext*> _fn_contexts;
 
-    /// Array access to fn_contexts_. Used by ScalarFnCall's codegen'd compute function
-    /// to access the correct FunctionContext.
-    /// TODO: revisit this
-    FunctionContext** _fn_contexts_ptr;
```

Comment:
may i ask how to use this ubsan tool?

Reply:
[p](https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html)